src/tiled/propertybrowser.cpp

```diff
@@ -618,6 +618,8 @@ void PropertyBrowser::addMapProperties()
 
     staggerIndexProperty->setAttribute(QLatin1String("enumNames"), mStaggerIndexNames);
 
+    addProperty(ParallaxOffsetProperty, QMetaType::QPointF, tr("Parallax Offset"), groupProperty);
```

Comment:
You also need to set the value of this property in `PropertyBrowser::updateProperties`.

Reply:
Done.